rules-reviewed/jboss-eap/weblogic/weblogic-webapp.windup.xml

```diff
@@ -162,8 +162,21 @@
                 <hint effort="1">
                     <message>Migrate to JBoss EAP 6: org.jboss.security.SecurityContextAssociation</message>
                     <link
-                        href="https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/6.3/html-single/API_Documentation/files/javadoc/org/jboss/security/SecurityContextAssociation.html"
-                        title="Security Context - JBoss 6" />
+                        href="https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/6.4/html-single/API_Documentation/files/javadoc/org/jboss/security/SecurityContextAssociation.html"
+                        title="Security Context - JBoss EAP 6" />
+                </hint>
+            </perform>
+        </rule>
+        <rule id="weblogic-webapp-09000">
+            <when>
+                <javaclass references="weblogic.servlet.security.ServletAuthentication"/>
+            </when>
+            <perform>
+                <hint effort="1" severity="mandatory" title="WebLogic proprietary annotation ServletAuthentication">
+                    <message>Migrate to Standard Java EE 6 Servlet Security: javax.servlet.annotation.ServletSecurity</message>
```

Comment:
Hey Marek, this is good, but hints should use complete sentences, and should probably describe both: a) the problem that was found and the intent of the code if possible, and b) the solution.